UPGRADE-3.x.md

```diff
@@ -1,6 +1,13 @@
 UPGRADE 3.x
 ===========
 
+UPGRADE FROM 3.x to 3.x
+=======================
+
+ ### Support for NelmioApiDocBundle > 3.6 is added
+
+ Since version 3.x, support for nelmio/api-doc-bundle > 3.6 is added. Controllers for NelmioApiDocBundle v2 was move to `Sonata\ClassificationBundle\Controller\Api\Legacy` and controllers for NelmioApiDocBundle v3 was added instead. If you extend them, you must change they using for correct Controller.
```

Comment:
```suggestion
 Since version 3.x, support for nelmio/api-doc-bundle > 3.6 is added. Controllers for NelmioApiDocBundle v2 were moved under `Sonata\ClassificationBundle\Controller\Api\Legacy\` namespace and controllers for NelmioApiDocBundle v3 was added as replacement. If you extend them, you must ensure they are using the corresponding inheritance.
```

Reply:
Current message was added in ClassificationBundle. I copy it. IMO it should be the same for all API bundles.

Reply:
I agree we must keep consistency, but the current message is not semantically right. That's why I suggested these changes.

Reply:
so I will update in in whole RPs